.github/workflows/trivy-scan-template.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: mkdir -p ballerina/lib
 
       - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@master
+        uses: aquasecurity/trivy-action@v0.23.0
```

Comment:
should this be trivy-action@0.23.0 ? without v?

Reply:
my bad. it seems their release release tag naming pattern is deviated from usual conventions. Fixed